### PR TITLE
Fix typos and improve wording

### PR DESCRIPTION
**Pull request by JohJakob**
- Fix typos
- Improve wording
- Use typographical apostrophes
- Use more precise project description in README
- Use level 2 heading for **License** in README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,25 +98,25 @@ <h5>16-bit Support and Color Management</h5>
...
-                <p>Gimel Studio's aim is not to neccessarily be a full replacement to all other graphics software, but to be a non-destructive 2d editor worthy to be a seamless suplement/alternative workflow to tools like Blender, GIMP, Krita, MyPaint, DarkTable, etc. Integration via a Blender addon, software file-format support, etc is planned.</p>
+                <p>Gimel Studio’s aim is not to neccessarily be a full replacement to all other graphics software, but to be a non-destructive 2D editor worthy to be a seamless suplement/alternative workflow to tools like Blender, GIMP, Krita, MyPaint, DarkTable, etc. Integration via a Blender add-on, software file format support, etc is planned.</p>
             </div>
             <div class="pure-u-1 pure-u-md-1-3">
                 <h5>GPU and CPU Processing</h5>
-                <p>We plan to use GLSL shaders to leverage GPU and CPU processing while being seamlessly cross-platform on Windows, Linux and MacOs (for 64-bit systems).</p>
+                <p>We plan to use GLSL shaders to leverage GPU and CPU processing while being seamlessly cross-platform on Windows, Linux and macOS (for 64-bit systems).</p>
```
Comment (Correct-Syntax) [inline index.html:105]: I am not sure whether it's suppose to be macOS, MacOs, macOs or what. It seems pretty confusing...  Shouldn't it be capitalized though?

**Review by Correct-Syntax — COMMENTED** [review]
You know what, I just realized my last commit probably reverted the changes I had made in the previous commit! I think I'll merge this PR then revert those changes.

File: index.html (hunk 1 above)
Comment (iwoithe) [reply index.html:105]: On [macOS](https://www.apple.com/macos/)'s website, the capitalisation is ``macOS``

Comment (JohJakob) [reply index.html:105]: Yes, macOS is correct.

(It’s in line with iOS, iPadOS, and watchOS now. It used to be OS X and Mac OS X before that.)

Comment (Correct-Syntax) [reply index.html:105]: Okay. I have seen it both ways and now I know the correct way. Thanks for the PR! 😄

**Pull request merged**